priv/repo/migrations/20211129171040_create_layouts.exs

```diff
@@ -6,7 +6,7 @@ defmodule Beacon.Repo.Migrations.CreateLayouts do
       add :id, :binary_id, primary_key: true
       add :site, :text
       add :title, :text
-      add :body, :text
+      add :template, :text
```

Comment:
Actually we can't change existing migrations :) Can you please create a new migration?

Reply:
Would this work?
https://github.com/BeaconCMS/beacon/pull/320/commits/0e046e883fdacab2bebb568d105c0cc839088ee1

Reply:
Yeah it does 👍🏻

Reply:
You mentioned "Adding a new migration to alter the column name in the table causes the older migration files to fail". Which error are you getting?

Reply:
```
Beacon.Repo.Migrations.MigrateLayoutsEventsSnapshots.up/0, at: priv/repo/migrations/20230626154010_migrate_layouts_events_snapshot.exs:32
** (Postgrex.Error) ERROR 42703 (undefined_column) column b0.template does not exist

    query: SELECT b0."id", b0."site", b0."title", b0."template", b0."meta_tags", b0."stylesheet_urls", b0."inserted_at", b0."updated_at" FROM "beacon_layouts" AS b0
    (ecto_sql 3.10.1) lib/ecto/adapters/sql.ex:913: Ecto.Adapters.SQL.raise_sql_call_error/1
    (ecto_sql 3.10.1) lib/ecto/adapters/sql.ex:828: Ecto.Adapters.SQL.execute/6
    (ecto 3.10.2) lib/ecto/repo/queryable.ex:229: Ecto.Repo.Queryable.execute/4
    (ecto 3.10.2) lib/ecto/repo/queryable.ex:19: Ecto.Repo.Queryable.all/3
    priv/repo/migrations/20230626154010_migrate_layouts_events_snapshot.exs:32: Beacon.Repo.Migrations.MigrateLayoutsEventsSnapshots.up/0
    (ecto_sql 3.10.1) lib/ecto/migration/runner.ex:289: Ecto.Migration.Runner.perform_operation/3
    (stdlib 4.3) timer.erl:235: :timer.tc/1
    (ecto_sql 3.10.1) lib/ecto/migration/runner.ex:25: Ecto.Migration.Runner.run/8

```

This migration fails on `mix setup`.
https://github.com/BeaconCMS/beacon/blob/main/priv/repo/migrations/20230626154010_migrate_layouts_events_snapshot.exs

I think it's because the `Beacon.Content.Layout` already has `template` in the field, but still `body` on the db.

Reply:
The issue is using a schema in the migration, it should be executing raw queries instead. I'll push a fix on `main`.

Reply:
Just pushed a fix to main - https://github.com/BeaconCMS/beacon/commit/129a130c978e589a0a0a6ea86116d652b9489b0e. Please rebase your branch and give it a try

Reply:
Thanks, it worked!